aviatrix/resource_aviatrix_firewall_instance.go

```diff
@@ -516,10 +528,21 @@ func resourceAviatrixFirewallInstanceRead(d *schema.ResourceData, meta interface
 			return fmt.Errorf("failed to set tags for firewall_instance on read: %v", err)
 		}
 	}
+	if fI.FirewallImageId != "" && intInSlice(cloudType, []int{goaviatrix.AWS, goaviatrix.AZURE}) {
```

Comment:
Should this be goaviatrix.IsCloudType(cloudtype, goaviatrix.AWSRelatedCloudTypes|goavtiatrix.AzureRelatedCloudTypes ?